xds/src/test/java/io/grpc/xds/XdsLbStateTest.java

```diff
@@ -217,83 +122,13 @@ public void onCompleted() {
     doReturn(channel).when(helper).createResolvingOobChannel(BALANCER_NAME);
   }
 
-  @After
-  public void tearDown() {
-    verifyNoMoreInteractions(random);
-  }
-
   @Test
   public void shutdownAndReleaseXdsCommsDoesShutdown() {
```

Comment:
This isn't really test something other than verifying a call to another method, which does not reveal any behavior. 
After I moved channel creation to `XdsLoadBalancer`, we can have a similar test that asserts the channel is not shutdown after calling `shutdownAnd...`. But don't change it here as it will introduce conflicts with #5836 .